src/server/cluster_family_test.cc

```diff
@@ -247,6 +247,102 @@ TEST_F(ClusterFamilyTest, ClusterConfigFullMultipleInstances) {
   // takes effect.
 }
 
+TEST_F(ClusterFamilyTest, ClusterGetSlotInfo) {
+  string node_id = Run({"dflycluster", "myid"}).GetString();
+  string config_prefix = R"json(
+      [
+        {
+          "slot_ranges": [
+            {
+              "start": 0,
+              "end": 16383
+            }
+          ],
+          "master": {
+            "id": ")json";
+  string config_suffix = R"json(",
+            "ip": "10.0.0.1",
+            "port": 7000
+          },
+          "replicas": []
+        }
+      ])json";
+  string full_config = absl::StrCat(config_prefix, node_id, config_suffix);
+
+  EXPECT_EQ(Run({"dflycluster", "config", full_config}), "OK");
+
+  Run({"debug", "populate", "100000"});
+
+  auto slots_info = Run({"dflycluster", "getslotinfo", "slots", "1", "2"}).GetVec();
+  EXPECT_EQ(slots_info.size(), 2);
+  auto slot1 = slots_info[0].GetVec();
+  EXPECT_EQ(slot1.size(), 3);
+  EXPECT_EQ(slot1[0], "1");
+  EXPECT_EQ(slot1[1], "key_count");
+  EXPECT_NE(slot1[2], "0");
+  auto slot2 = slots_info[1].GetVec();
```

Comment:
I think that you could replace all of these lines with:
`EXPECT_THAT(slots_info, ElementsAre("1", "key_count", "0"));`

Reply:
I am checking that slot1[2] does not equals "0", this is after the populate so there is data in the slot.
I dont know what is the number of elements in the slot as I use debug populate and dont know how many elements are in each slot, just want to make sure that there are
I can use this for the checks when I expect it to be 0

Reply:
Sorry, I missed the fact that it's a not. How about:

```cpp
EXPECT_THAT(slots_info, ElementsAre("1", "key_count", Not("0")));
```

?

Reply:
I am thinking to introduce a debug populate command which will get slot range and will populate keys only in this slot.
Then I will be able to use this command here and check the exact number of keys in the slot

Reply:
EXPECT_THAT(slots_info, ElementsAre("1", "key_count", Not("0")));
This looks good I will fix

Reply:
> I am thinking to introduce a debug populate command which will get slot range and will populate keys only in this slot.

Since we already expose the function that returns the slot for a given key, you _could_ do it at the test, or as a test function.